nautobot/core/api/metadata.py

```diff
@@ -304,9 +310,47 @@ def determine_metadata(self, request, view):
             )
 
             metadata["view_options"] = self.determine_view_options(request, serializer)
+            metadata["filters"] = self.get_filter_info(view)
```

Comment:
This probably doesn't need to be in the block `if hasattr(view, "get_serializer")`? The `get_filter_info` method below checks for the `filterset_class` attr on the view and returns early if it's not present.